prep-javascript-functions/main.js

```diff
@@ -0,0 +1,62 @@
+function addTwoNumbers(x, y) {
+  return x + y;
+}
+
+var sum = addTwoNumbers(4, 4);
+console.log('sum', sum);
+
+function converHoursToMinutes(x, y) {
+  return x * y;
+}
+
+var minutes = converHoursToMinutes(2, 60);
+console.log('minutes', minutes);
+
+function getGreeting(World) {
+  return ('Hello World!');
```

Comment:
Here you should be using your parameter to greet the name passed into the function. Parameter name should be something like `name` then in the function you should be returning something like: `'Hello ' + name;`